### ISSUE TITLE
Propose renaming package to "malariagen_cloud"

### ISSUE THREAD

**Issue by alimanfoo** [malariagen/malariagen-data-python#288]
I'd like to suggest we rename this package from "malariagen_data" to "malariagen_cloud". Why? I think it would be a little easier to explain if we were able to talk about the "MalariaGEN cloud API" which provides an interface to data stored in the cloud. It also might be less confusing, as the current name "malariagen_data" suggests that it is a package containing actual data, rather than a package providing an API to data stored in the cloud. 

In other words, we could explain that the "MalariaGEN cloud data store" is a store of data in the cloud, and this can be accessed via the "MalariaGEN cloud API".

**Comment by sanjaynagi** [comment]
Interesting! I have no strong opinion, but agree that it would make it clearer and less confusing as to what the package actually does. It also sounds a bit fancier and more advanced, which may be beneficial when interacting with, for example, funders.

**Comment by cclarkson** [comment]
Like it, makes it sounds like we live in the future.

**Comment by alimanfoo** [comment]
Thanks chaps for the feedback. If we do make the change, we'll have to update all the training materials. Also the videos will be then out of date, but maybe that's OK if we add a disclaimer alongside the video that the name of the package has changed. Will mull it over for a bit I think.

**Comment by alimanfoo** [comment]
Cc @kathryn1995, @leehart, @jonbrenas, @podpearson, I'm thinking of renaming the package from malariagen_data to malariagen_cloud, any comments welcome.

**Comment by leehart** [comment]
How about `malariagen` as the package name, and maybe `cloud` as a sub-package name?

**Comment by leehart** [comment]
The term `cloud` might imply access to other things associated with the cloud, such as cloud compute.
I gather the original intention of using the term `data` was to focus on providing access to the data, for data analysis. I suppose, to some extent, users will need to download the data from the cloud, at which point it isn't really `cloud` data any more (Is it `rain`?!), but we might provide tools for analysing that cloud-sourced data.

**Comment by lcerdeira** [comment]
@alimanfoo makes sense to change the name, but I believe it would be better to remove the word' data'. I agree with @leehart to keep only malariagen "the term cloud might imply access to other things associated with the cloud, such as cloud computing".

**Comment by alimanfoo** [comment]
> I gather the original intention of using the term `data` was to focus on providing access to the data, for data analysis. I suppose, to some extent, users will need to download the data from the cloud, at which point it isn't really `cloud` data any more (Is it `rain`?!), but we might provide tools for analysing that cloud-sourced data.

Yes original intention was to convey it's an API for accessing data. Although the main aim of the package is to enable cloud-native data analysis, i.e., analysis of data using cloud computing resources with good data proximity, without needing to manually download anything. Not sure where that leaves us re naming :)

**Comment by leehart** [comment]
Just to re-ignite this thread, in the name of resolution, may I propose something that seems relatively easy and low impact:
1. Publish a new package named simply `malariagen` (I believe this is available, but might not be for much longer!)
2. `malariagen` will include a subpackage named `cloud`, so that `malariagen.cloud` does the same as the current `malariagen_data` package.
3. `malariagen` will also include a subpackage named `data`, so that `malariagen.data` does the same as the current `malariagen_data` package, effectively being an alias for the `malariagen.cloud` subpackage (or vice versa).
4. Existing code can be update to use `malariagen.data`, which might be deprecated, or `malariagen.cloud`, which might become the preferred alias moving forward. (Who knows!)
5. We might allow a schism whereby `cloud` develops into software more clearly associated with the cloud storage and services, such as download data or cloud computing, whereas `data` develops into software more clearly associated with data that are already on disk or in memory. (Or we might nip that in the bud by only discontinuing `mgen.data`.)
5. We then refer to the new `malariagen` package as the MalariaGEN API or the malariagen python package, through which one can download and analyse data published by MalariaGEN (and partners? or anyone?), amongst other things (potentially).

Personally, since some of the existing classes are very monolithic, I would also like to see things like plotting functions have their own namespace, but things like that can be done later as separate issues, of course.

**Comment by leehart** [comment]
I parked the package name `malariagen` to avoid the inconvenience of anyone else taking it, and I made a proof-of-concept:
- https://colab.research.google.com/drive/1nQXZFuc5sgYWyV3JzghUvTxyRHfB_E3y?usp=sharing

**Comment by alimanfoo** [comment]
> 6\. We then refer to the new `malariagen` package as the "MalariaGEN API" or the "malariagen python package", through which one can download and analyse data published by MalariaGEN (and partners? or anyone?), amongst other things (potentially).

I like this FWIW.

**Comment by leehart** [comment]
@alimanfoo I recently tried adding you as a co-owner of https://pypi.org/project/malariagen/ but it looks like the invite expired.

Maybe we should decide what we're going to do here, if anything.
